piety-promoted-v3/modernized/15-chapter-15.adoc

```diff
@@ -251,12 +251,12 @@ after some personal abuse, by a persecuting alderman,
 in consequence of being taken up at the Bull-and-Mouth meeting.
 The prison was then crowded with Friends,
 and the noisome confinement probably occasioned not only this Friend`'s death,
-out that of his fellow prisoner, Edward Burrough.
+but that of his fellow prisoner, Edward Burrough.
```

Comment:
It looks like in the original (and reads better) that the top of the "b" was cut off.